exercises/concept/little-sisters-vocab/.docs/instructions.md

```diff
@@ -0,0 +1,69 @@
+# Instructions
+
+You're helping your younger sister with her English vocabulary homework, which she's finding very tedious. Her class is learning to create new words by adding _prefixes_ and _suffixes_, and have been given different sets of words to modify. The teacher is looking for correctly transformed words and correct spelling, given the word beginning or ending.
+
+There's four activities in the assignment, each with a set of text or words to work with.
+
+## 1. Add a prefix to a word
+
+One of the most common prefixes in English is `un`, meaning "not". In this activity, your sister needs to make negative, or "not" words by adding `un` to them.
+
+Implement the `add_prefix_un()` function that takes `word` as a parameter and returns a new `un` prefixed word:
+
+```python
+>>> add_prefix_un("happy")
+'unhappy'
+
+>>> add_prefix("manageable")
```

Comment:
Not sure whether this should be `add_prefix_un` or line 14 should be `add_prefix`, but to be consistent:
```suggestion
>>> add_prefix_un("manageable")
```